cryptotik/binance.py

```diff
@@ -39,17 +39,6 @@ def api(cls, url, params):
         except requests.exceptions.RequestException as e:
             raise APIError(e)
 
-    def get_nonce(self):
```

Comment:
Why is there a binance change inserted here?

Reply:
Realized the method was not needed, and was inserted by mistake. Didn't bother to commit it separately. Sorry!

Reply:
Binance API does not need nonce?

Reply:
no, it uses timestamp instead.

Reply:
Okay then.